app/react/Metadata/components/MetadataForm.js

```diff
@@ -50,6 +51,15 @@ export class MetadataForm extends Component {
 
   renderTemplateSelect(templateOptions, template) {
     if (templateOptions.size) {
+      const templateId = template.get('_id');
+      let sortedTemplates = templateOptions
+        .toJS()
+        .filter(defTemp => defTemp.value !== templateId)
+        .sort((a, b) => a.label > b.label);
+
+      const defTemplate = templateOptions.toJS().filter(temp => temp.value === templateId);
+      sortedTemplates = [...defTemplate, ...sortedTemplates];
```

Comment:
Same question as in the other case: did we decide to treat the selected template specially?

Reply:
Removed